Makefile

```diff
@@ -94,7 +94,16 @@ check-go-format: prebuild-check
 	&& exit 1 \
 	|| true
 
-
+.PHONY: install-git-hooks
+## Creates useful git hooks (e.g. pre-push go-format-code check)
+install-git-hooks:
+	ln -sfv ../../.pre-push .git/hooks/pre-push
```

Comment:
@hectorj2f no it wasn't it is a pre push format checking of go code. I don't have to name the exact make target because it is not important.